lib/tasks/dev.rake

```diff
@@ -3,48 +3,72 @@ namespace :dev do
   require 'faker'
 
   desc 'Generate test data'
-  task :generate_data, [:developers_num, :language_num, :programming_language_num] => :environment do |_, args|
-    set_args(args)
+  task :generate_data, %i[developers_num languages_num programming_languages_num] => :environment do |_, args|
+    process_args(args)
 
-    developers = []
-    languages = []
-    programming_languages = []
+    @list_developer = []
+    @list_language = []
+    @list_programming_language = []
 
-    @args[:developers_num].times do
-      email = loop do
-        email = Faker::Internet.email
-        break email unless Developer.exists?(email: email)
-      end
+    puts '========Start generate test data, params========'
+    puts 'Number of developer: ' + @args[:developers_num].to_s
+    puts 'Number of language: ' + @args[:languages_num].to_s
+    puts 'Number of programming language: ' + @args[:programming_languages_num].to_s

```

Comment:
Please read the style guide (not only this part if you haven't read it through).
https://github.com/bbatsov/ruby-style-guide#strings

Also the one for rails.
https://github.com/bbatsov/rails-style-guide

There're the Vietnamese versions.